extras/userdoc/md/documentation/part-2-populations-of-neurons.md

```diff
@@ -230,30 +270,46 @@ These are the new functions we introduced for the examples in this handout.
 
 ### Getting information about NEST
 
-`PrintNetwork(depth=1, subnet=None)`:  
-Print the network tree up to `depth`, starting at `subnet`. If `subnet` is
-omitted, the current sub-network is used instead.
+### Getting and setting basic settings and parameters of NEST
+
+-   `GetKernelStatus(keys=none)`:
+
+    Obtain parameters of the simulation kernel.
+    Returns:
```

Comment:
Maybe a compact table/list of methods would be useful, leaving all details to the docstrings.